LocalReferenceLinkRel/explainer.md

```diff
@@ -148,6 +148,81 @@ the shadow root where they are defined, as illustrated by the following examples
   Styles defined inside the sibling Shadow Root are not applied, so "Inside Sibling Shadow DOM" is not blue.
 </p>
 
+### Fetch Behavior
+
+In user agents where "Local References In `<link>` Tags" is not supported,
```

Comment:
I'm wondering if the quotes here are necessary? I think it would read fine without that (and even removing capitalization of that is probably fine in the context of this sentence)